tests/session/test_subscription.py

```diff
@@ -558,3 +559,43 @@ async def test_unsubscribeIsAtomic(websocket, context_id, iframe_id):
         'method': 'log.entryAdded',
         'params': AnyWithEntries({'text': 'SOME_MESSAGE'})
     }) == resp
+
+
+@pytest.mark.asyncio
+async def test_unsubscribe_from_detached_target(websocket, context_id,
```

Comment:
Please get the updated version of the test: https://github.com/GoogleChromeLabs/chromium-bidi/pull/2606/files#diff-33a034f6cd8ec2ac450e990aa128abd652b00fd7717e778889772b4bb1daac3eR565